tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/DocumentType/DocumentType.spec.ts

```diff
@@ -0,0 +1,765 @@
+      await umbracoUi.documentType.isSuccessNotificationVisible();
+      const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+      expect(documentTypeData.variesByCulture).toBeTruthy();
+    });
+
+    test('can add allow segmentation for a document type', async ({umbracoApi, umbracoUi}) => {
+      // Arrange
+      await umbracoApi.documentType.createDefaultDocumentType(documentTypeName);
+      await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+
+      // Act
+      await umbracoUi.documentType.goToDocumentType(documentTypeName);
+      await umbracoUi.documentType.clickDocumentTypeSettingsTab();
+      await umbracoUi.documentType.clickVaryBySegmentsButton();
+      await umbracoUi.documentType.clickSaveButton();
+
+      // Assert
+      await umbracoUi.documentType.isSuccessNotificationVisible();
+      const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+      expect(documentTypeData.variesBySegment).toBeTruthy();
+    });
+
+    test('can set is an element type for a document type @smoke', async ({umbracoApi, umbracoUi}) => {
+      // Arrange
+      await umbracoApi.documentType.createDefaultDocumentType(documentTypeName);
+      await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+
+      // Act
+      await umbracoUi.documentType.goToDocumentType(documentTypeName);
+      await umbracoUi.documentType.clickDocumentTypeSettingsTab();
+      await umbracoUi.documentType.clickTextButtonWithName('Element type');
+      await umbracoUi.documentType.clickSaveButton();
+
+      // Assert
+      await umbracoUi.documentType.isSuccessNotificationVisible();
+      const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+      expect(documentTypeData.isElement).toBeTruthy();
+    });
+
+    // TODO: Unskip. Currently The cleanup is not updated upon save
+    test.skip('can disable history cleanup for a document type', async ({page, umbracoApi, umbracoUi}) => {
+      // Arrange
+      await umbracoApi.documentType.createDefaultDocumentType(documentTypeName);
+      await umbracoUi.documentType.goToSection(ConstantHelper.sections.settings);
+
+      // Act
+      await umbracoUi.documentType.goToDocumentType(documentTypeName);
+      // Is needed
+      await umbracoUi.waitForTimeout(200);
+      await page.locator('umb-body-layout').getByRole('tab', {name: 'Settings'}).click({force: true});
+      await page.getByText('Auto cleanup').click();
+      await umbracoUi.documentType.clickSaveButton();
+
+      // Assert
+      await umbracoUi.documentType.isSuccessNotificationVisible();
+      const documentTypeData = await umbracoApi.documentType.getByName(documentTypeName);
+      expect(documentTypeData.cleanup.preventCleanup).toBeTruthy();
+    });
+  });
+
+  test.describe('Templates tab', () => {
+    test('can add an allowed template to a document type @smoke', async ({umbracoApi, umbracoUi}) => {
```

Comment:
Missing the test that sets a template as the default. You can leave it as a TODO.